aws/ec2/variables.tf

```
@@ -25,6 +25,7 @@ variable "ebs" {
     device_name = string
     az          = string
     type        = string
+    iops        = string
```

```suggestion
    iops        = optional(string)
```

This way potential module clients won't get, out of surprise, a new iops paratmeter

---

ok noted.  Makes sense. I will emend and request a new PR.